firebird/aardvark.py

```diff
@@ -5,60 +5,71 @@
 def pyccd_tile_spec_queries(url):
     """
     A map of pyccd spectra to tile-spec queries
-    Args:
-        url: full url for tile-spec endpoint
-             http://localhost:9200/landsat/tile-specs
+    :param url: full url for tile-spec endpoint
+    :return: map of spectra to tile spec queries
+    :example:
+    >>> pyccd_tile_spec_queries('http://localhost:9200/landsat/tile-specs')
+    {"red":     'http://localhost:9200/landsat/tile-specs?q=tags:red AND sr',
+     "green":   'http://localhost:9200/landsat/tile-specs?q=tags:green AND sr'
+     "blue":    'http://localhost:9200/landsat/tile-specs?q=tags:blue AND sr'
+     "nir":     'http://localhost:9200/landsat/tile-specs?q=tags:nir AND sr'
+     "swir1":   'http://localhost:9200/landsat/tile-specs?q=tags:swir1 AND sr'
+     "swir2":   'http://localhost:9200/landsat/tile-specs?q=tags:swir2 AND sr'
+     "thermal": 'http://localhost:9200/landsat/tile-specs?q=tags:thermal AND toa'
+     "cfmask":  'http://localhost:9200/landsat/tile-specs?q=tags:cfmask AND sr'}
     """
-    return {"red": ''.join([url, '?q=tags:red AND sr']),
-            "green": ''.join([url, '?q=tags:green AND sr']),
-            "blue": ''.join([url, '?q=tags:blue AND sr']),
-            "nir": ''.join([url, '?q=tags:nir AND sr']),
-            "swir1": ''.join([url, '?q=tags:swir1 AND sr']),
-            "swir2": ''.join([url, '?q=tags:swir2 AND sr']),
+    return {"red":     ''.join([url, '?q=tags:red AND sr']),
+            "green":   ''.join([url, '?q=tags:green AND sr']),
+            "blue":    ''.join([url, '?q=tags:blue AND sr']),
+            "nir":     ''.join([url, '?q=tags:nir AND sr']),
+            "swir1":   ''.join([url, '?q=tags:swir1 AND sr']),
+            "swir2":   ''.join([url, '?q=tags:swir2 AND sr']),
             "thermal": ''.join([url, '?q=tags:thermal AND toa']),
-            "cfmask": ''.join([url, '?q=tags:cfmask AND sr'])}
+            "cfmask":  ''.join([url, '?q=tags:cfmask AND sr'])}
 
 
 def chip_specs(query):
     """
     Queries elasticsearch and returns chip_specs
-    Args:
-        query: full url query for elasticsearch
-               http://localhost:9200/landsat/tile-specs?q=tags:red AND sr
-    Returns:
-        ['chip_spec_1', 'chip_spec_2', ...]
+    :param query: full url query for elasticsearch
+    :returns: sequence of chip specs
+    :example:
+    >>> chip_specs('http://localhost:9200/landsat/tile-specs?q=tags:red AND sr')
+    ['chip_spec_1', 'chip_spec_2', ...]
     """
     js = requests.get(query).json()
-    if js:
-        return [i for i in js if 'ubid' in i]
+    if 'hits' in js and 'hits' in js['hits']:
```

Comment:
i yanked this in my PR b/c i'd never seen the 'hits' key in the response from a *-specs query, assuming its planned in the future?